systems/plants/RigidBodyVisualizer.m

```diff
@@ -40,6 +45,53 @@ function inspector(obj,x0,state_dims,minrange,maxrange,options)
       
       inspector@Visualizer(obj,x0,state_dims,minrange,maxrange,obj.model);
     end
+    
+    function draw(obj,t,x)
+      n = obj.model.num_positions;
+      q = x(1:n); %qd=x(n+(1:n));
+      kinsol = obj.model.doKinematics(q);
+      
+      % for debugging:
+      %co = get(gca,'ColorOrder');
+      %h = [];
+      % end debugging
+
+      for i=1:length(obj.model.body)
```

Comment:
I'd use `numel` rather than `length` here. That would make this code robust if we later decide (for some unknown reason) to store the geometries in a two dimensional cell array.